WGS_pipeline/vep_sv.wdl

```diff
@@ -5,12 +5,12 @@ workflow VEP_SV_Workflow {
         File input_vcf
         String sample_id
 
-        File vep_data = "gs://cds-vep-data/homo_sapiens_vep_110_GRCh38.tar.gz"
+        File vep_data = "gs://cds-vep-data/homo_sapiens_vep_112_GRCh38.tar.gz"
         File fasta = "gs://cds-vep-data/Homo_sapiens_assembly38.fasta.gz"
         File fai = "gs://cds-vep-data/Homo_sapiens_assembly38.fasta.gz.fai"
         File gzi = "gs://cds-vep-data/Homo_sapiens_assembly38.fasta.gz.gzi"
-        File gnomad = "gs://gcp-public-data--gnomad/papers/2019-sv/gnomad_v2.1_sv.sites.vcf.gz"
-        File gnomad_idx = "gs://gcp-public-data--gnomad/papers/2019-sv/gnomad_v2.1_sv.sites.vcf.gz.tbi"
+        File gnomad = "gs://ccleparams/gnomad.v4.1.sv.sites.pass.vcf.gz"
```

Comment:
I assume this is a filtered version of `gs://gcp-public-data--gnomad/release/4.1/genome_sv/gnomad.v4.1.sv.sites.vcf.gz`, right? If so, is there a script in this repo to regenerate our version for future gnomad releases?

Reply:
Yes! It's 3 lines of code. Would it make more sense for me to leave that as a comment in this wdl or save it as a separate file somewhere?

Reply:
Comment sounds fine in that case

Reply:
done!